torch/_functorch/aot_autograd.py

```diff
@@ -1858,22 +1858,39 @@ def backward(ctx, *flat_args):
                 list(ctx.symints) + list(ctx.saved_tensors) + list(contiguous_args)
             )
             del contiguous_args
-            if CompiledFunction.compiled_bw is None:
-                # TODO - pass in fake tensors ?
-                context = disable_autocast_manager if disable_amp else nullcontext
-                with context(), track_graph_compiling(aot_config, "backward"):
-                    CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                        bw_module, all_args
+
+            class CustomDelayedError(torch.autograd.Function):
```

Comment:
AOTAutogradCompiledBackward ?

Reply:
Sounds good. Two questions/comments:

1. Should we consider the names of these nodes public API (I think so?)
2. I like having "AOTAutograd" in the node name since it is more informative. It feels bad though that the other custom Function is "CompiledFunction". 

I'm okay with landing it like this, but I'm curious if there's anything to do to improve this situation.